packages/govuk-frontend-review/browsersync.config.js

```diff
@@ -0,0 +1,39 @@
+const { join } = require('path')
+
+const { paths, ports } = require('govuk-frontend-config')
+
+/**
+ * Browsersync config
+ *
+ * @type {import('browser-sync').Options}
+ */
+module.exports = {
+  proxy: `http://localhost:${ports.app}`,
+
+  // Prevent browser mirroring
+  ghostMode: false,
+
+  // Prevent browser opening
+  open: false,
+
+  // Allow for Express.js restart
+  reloadDelay: 1000,
+
+  // Files to watch for live reload
+  files: [
```

Comment:
@domoscargin Do we need to watch and reload for `*.njk` files too?

Reply:
Ooh, yeah, that'd be good.

Reply:
Works really well, seamless!